corehq/apps/users/models.py

```diff
@@ -670,7 +670,7 @@ def role_label(self, domain=None):
         except TypeError:
             return "Unknown User"
         except DomainMembershipError:
-            return "Unauthorized User"
+            return "Dimagi User" if self.is_global_admin() else "Unauthorized User"
```

Comment:
i know you didn't write the original code, but i think these should all be translated